### PR TITLE
Adding a notice about the AzureRM vs AzureSM Providers

### PR DESCRIPTION
**Pull request by tombuildsstuff**
Adding a notice about the AzureRM vs AzureSM Providers to the Readme and the Github Issues Template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +25,13 @@ Clone repository to: `$GOPATH/src/github.com/hashicorp/terraform-provider-$PROVI
 
 ```sh
 $ mkdir -p $GOPATH/src/github.com/hashicorp; cd $GOPATH/src/github.com/hashicorp
-$ git clone git@github.com:hashicorp/terraform-provider-$PROVIDER_NAME
+$ git clone git@github.com:hashicorp/terraform-provider-azure
```
Comment (grubernaut) [inline README.md]: While you're here, might be good to change `hashicorp` to `terraform-providers`, and same below.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ Terraform Provider
 
 ![Terraform](https://rawgithub.com/hashicorp/terraform/master/website/source/assets/images/logo-hashicorp.svg)
 
+Please Note
+------------
+
+This repository is for the older Azure Provider (also known as Service Management / "the old portal") which Microsoft recommend only for existing requirements. For new requirements, the Resource Manager / "new portal" is recommended - which is available in [the AzureRM Terraform Provider](https://github.com/terraform-provider/terraform-provider-azurerm).
```
Comment (grubernaut) [inline README.md]: ~terraform-provider~ `terraform-providers`, and same above

**Review by grubernaut — APPROVED** [review]
LGTM, minor nits though. Ship when updated :smile:

File: README.md (hunk 2 above)
Comment (tombuildsstuff) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (tombuildsstuff) [reply README.md]: Done. Fixed the logo too

**Pull request merged**